controllers/watcherapplier_controller.go

```diff
@@ -207,7 +219,10 @@ func (r *WatcherApplierReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 
 	instance.Status.Conditions.MarkTrue(condition.ServiceConfigReadyCondition, condition.ServiceConfigReadyMessage)
 
-	// TODO(dviroel): Add create Deployment/StatefulSet
+	result, err = r.ensureDeployment(ctx, helper, instance, inputHash, map[string]string{})
```

Comment:
We are actually not using annotations at all and hardcodding empty list. We could simply remove all the path for them from here up to ensureDeployment definition and watcherapplier/statefulset.go.

Reply:
agree, this is just adding noise for now

Reply:
Do we have use case for NetworkAttachments? Note that the pods can reach both prometheus and the rest of openstack API services via pods network even in isolated networks. The only case I see we may need it is if we need the pods to reach some networking endpoint not exposed via openshift services/routes. Do we have any of those cases?

Reply:
ok, so lets move annotations part to the network attachments PR.